Alignment/OfflineValidation/interface/PreparePVTrends.h

```diff
@@ -0,0 +1,300 @@
+#ifndef ALIGNMENT_OFFLINEVALIDATION_PREPAREPVTRENDS_H_
+#define ALIGNMENT_OFFLINEVALIDATION_PREPAREPVTRENDS_H_
+
+#include "TArrow.h"
+#include "TAxis.h"
+#include "TCanvas.h"
+#include "TF1.h"
+#include "TFile.h"
+#include "TGaxis.h"
+#include "TGraph.h"
+#include "TGraphAsymmErrors.h"
+#include "TGraphErrors.h"
+#include "TH1F.h"
+#include "TH2F.h"
+#include "TLatex.h"
+#include "TLegend.h"
+#include "TList.h"
+#include "TMath.h"
+#include "TObjArray.h"
+#include "TObjString.h"
+#include "TPad.h"
+#include "TPaveText.h"
+#include "TProcPool.h"
+#include "TProfile.h"
+#include "TROOT.h"
+#include "TStyle.h"
+#include "TSystem.h"
+#include "TSystemDirectory.h"
+#include "TSystemFile.h"
+#include <TStopwatch.h>
+#include <algorithm>
+#include <bitset>
+#include <fstream>
+#include <functional>
+#include <iostream>
+#include <iterator>
+#include <map>
+#include <sstream>
+#include <vector>
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/json_parser.hpp>
+
+#include "Alignment/OfflineValidation/interface/OutPVtrends.h"
+
+/*!
+ * \def some convenient I/O
+ */
+#define logInfo std::cout << "INFO: "
+#define logWarning std::cout << "WARNING: "
+#define logError std::cout << "ERROR!!! "
```

Comment:
Wouldnt it have been easier to just used the MsgLogger variants of this?

Reply:
in the original implementation this was part of a ROOT macro which was supposed to work also without a cms environment. Given this is now in `interface` (even though it not very clear to me why it should be the case....) and so by construction implied to be distributed with `cmssw`, it probably makes sense to change it to MsgLogger.